samples/browser/src/main/java/org/mozilla/samples/browser/request/SampleUrlEncodedRequestInterceptor.kt

```diff
@@ -0,0 +1,42 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package org.mozilla.samples.browser.request
+
+import android.content.Context
+import mozilla.components.browser.errorpages.ErrorPages
+import mozilla.components.browser.errorpages.ErrorType
+import mozilla.components.concept.engine.EngineSession
+import mozilla.components.concept.engine.request.RequestInterceptor
+import mozilla.components.concept.engine.request.RequestInterceptor.InterceptionResponse
+import mozilla.components.concept.engine.request.RequestInterceptor.ErrorResponse
+import org.mozilla.samples.browser.ext.components
+
+/**
+ * Example of a request interceptor that loads error pages with URL encoding (images)
```

Comment:
I'm not sure if this is necessary but I thought it might be nice for consumers to have an example to look at here. Thoughts @jonalmeida?

Reply:
In the normal `SampleRequestInterceptor`, you mean?

Reply:
I mean put the entire sample inside comments for `RequestInterceptor`

Reply:
Actually I just realized this is inside sample browser 🤦‍♂ Nevermind, feel free to leave this